src/components/NoteDisplay/DeleteMode.js

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from "react";
+import Button from "../shared/Button";
+import styled from "styled-components";
+import { AppContext } from "../../contexts/AppContext";
+import globalTypes from "../../GlobalTypes";
+import { useHistory } from "react-router-dom";
+
+const CustomButton = styled(Button)`
+    position: relative;
+
+    &::before {
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        display: block;
+        background-color: rgba(0, 0, 0, 0.2);
+        top: 0;
+        left: 0;
+        opacity: 0;
+    }
+
+    &:hover::before {
+        opacity: 1;
+    }
+`;
+
+export default function DeleteMode({ noteId }) {
+    const { state, dispatch } = useContext(AppContext);
+    const history = useHistory();
+
+    return (
+        <>
+            <Button></Button>
```

Comment:
For components that doesn't have any children, you can do this instead:
```
<Button />
```

Tho may I ask what this button is for?

Reply:
Placeholder lang po para mapush niya sa right ung ibang buttons . Dapat pala flex-end na lang ginawa ko XD

Reply:
Ow sige sige hahaha.